musmart/melody/contour/step_contour.py

```diff
@@ -0,0 +1,217 @@
+"""Calculates the Step Contour of a melody, along with related features, as implemented
+in the FANTASTIC toolbox of Müllensiefen (2009) [1].
+
```

Comment:
Does Müllensiefen provide any reference for the original source of this algorithm?

Reply:
cited now